api_gateway/views/recomendation.py

```diff
@@ -25,9 +29,13 @@
 @returns_json
 def get_list(user_id: int):
     try:
-        response = GRPC_STUB.List(recomendations_pb2.RecomendationsRequest(
+        book_ids = GRPC_RECOMENDATOR_STUB.List(recomendations_pb2.RecomendationsRequest(
             id=user_id
         ))
+        response = [
+            GRPC_CATALOG_STUB.Retrieve(book_pb2.BookRetrieveRequest(id=book_id.id))
```

Comment:
Is this better than making only one grpc query with all the ids?

Reply:
Is that possible?  I only see request for retrieving all books or just one

Reply:
Well, we could add it, I'm not saying that it's already implemented. But if we are running out of time it's ok

Reply:
I think I could implement it today, lets talk it on the daily